File: java/update_rangefill.java
HttpResponse<String> response = Unirest.patch("https://graph.microsoft.com/v1.0/me/drive/items/%7Bid%7D/workbook/names(%3Cname%3E)/range/format/fill")
  .header("content-type", "application/json")
  .body("{ }")
  .asString();

File: python/update_rangefill.py
import http.client

conn = http.client.HTTPSConnection("graph.microsoft.com")

payload = "{ }"

headers = { 'content-type': "application/json" }

conn.request("PATCH", "/v1.0/me/drive/items/%7Bid%7D/workbook/names(%3Cname%3E)/range/format/fill", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))
